src/test/java/org/eclipse/microprofile/starter/core/model/JessieMavenTest.java

```diff
@@ -0,0 +1,17 @@
+package org.eclipse.microprofile.starter.core.model;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
```

Comment:
CheckStyle fails on star imports.

instead of `assertEquals` you can use `Assert.assertEquals` and use a regular import